### ISSUE TITLE
Clear preferences?

### ISSUE THREAD

**Issue by cars10** [AndyBarron/preferences-rs#21]
Is there a way to clear / remove the saved preferences?

**Comment by cars10** [comment]
For now i overwrite them with an empty collection, but sometimes i would like to do a "hard reset".

**Comment by ishitatsuyuki** [comment]
Agree. There should be a method to delete an item (file) from the hierarchy.

**Comment by Boscop** [comment]
Yes, please!
